.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
Please omit, would rather avoid having a file just for this folder since it should be in a global ignore file.